language/move-lang/functional-tests/tests/0L/gas/gas_transfer_check.move

```diff
@@ -17,7 +17,7 @@ fun main(account: &signer) {
 }
 }
 ////////// Transfers should fail ////////
-// check: "Keep(ABORTED { code: 1544,"
```

Comment:
@sm86 Are we able to the error code in these failures?

Reply:
Updated, thanks!